clientbuffer.cpp

```diff
@@ -52,9 +52,10 @@ class CClientBufferMod : public CModule
     MODCONSTRUCTOR(CClientBufferMod)
     {
         AddHelpCommand();
-        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier>", "Add a client.");
+        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier> [timelimit]", "Add a client with optional time limit.");
         AddCommand("DelClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnDelClientCommand), "<identifier>", "Delete a client.");
         AddCommand("ListClients", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnListClientsCommand), "", "List known clients.");
+        AddCommand("SetClientTimeLimit", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnSetClientTimeLimit), "<identifier> [timelimit]", "Change a clients time limit.");           
```

Comment:
- I think it doesn't make sense for `timelimit` to be optional here, so it should be specified as `<timelimit>` in the command description.
- Typo (clients -> client's)

```suggestion
        AddCommand("SetClientTimeLimit", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnSetClientTimeLimit), "<identifier> <timelimit>", "Change a client's time limit.");           
```